### ISSUE TITLE
Baseline validation set 구축

### ISSUE THREAD

**Issue by jeongjae96** [jeongjae96/synthesis-car-od#2]
## Description
- 현재 baseline 코드는 validation이 없다. 
- 검증셋을 만들어, 성능 평가를 할 수 있어야 한다.
- 평가 지표 mAP@0.85로 성능이 기록되어야 한다.

## Tasks
- [ ] train/val split
- [ ] train/val loss logging
- [ ] train/val map@0.85 logging
- [ ] wandb logging
